src/components/Restaurant.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+
+const Order = ({ orderType }) => {
```

Comment:
This is great separation making them all into smaller components however it still might be best to split them into separate files to keep them smaller and easier to read.